app/src/main/res/layout-land/app_version_fragment.xml

```diff
@@ -15,34 +15,26 @@
 
     <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
-      android:layout_height="match_parent">
+      android:layout_height="match_parent"
+      android:background="@color/profileEditBackground">
```

Comment:
Avoid using background colors that are specific to files. Create new generic background and assign it here or create file specific background color. 
It may create problem in future if profileEditBackground's color code is changed.

Reply:
+1

Reply:
@veena14cs  included a new color for it